pandas/tests/reshape/concat/test_index.py

```diff
@@ -257,3 +257,11 @@ def test_concat_multiindex_dfs_with_deepcopy(self):
         tm.assert_frame_equal(result_copy, expected)
         result_no_copy = concat(example_dict, names=["testname"])
         tm.assert_frame_equal(result_no_copy, expected)
+
+    def test_concat_multiindex_unique(self):
+        # GH#44786
+        df1 = DataFrame({"col": ["a", "b", "c"]}, index=["1", "2", "2"])
+        df2 = concat([df1], keys=["X"])
+        result = np.array([df2.index.is_unique])
```

Comment:
please call df2 result 

You can simply Use assert to check for uniqueness 
Pleasd also check the duplicate attribute

Reply:
Thank you for the suggestions! Just resolved them